src/partials/_sidebar.module.scss

```diff
@@ -1,9 +1,10 @@
 @use "./variables" as *;
 
 .sidebar {
-  border: $lines-light 2px solid;
-  width: 200px;
+  border: $lines-light 1px solid;
+  width: 300px;
```

Comment:
this is a container, this would be a good place to define width for different breakpoints 👍